docs/_layouts/post.html

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Did you change these theme files? For the ones you didn't, is it possible to point to copies provided by GitHub instead of hosting our own copy?

Reply:
They have been changed a bit but since I've refactored the files to just use the page template we can cut the other ones out.